### ISSUE TITLE
error when rejecting old token

### ISSUE THREAD

**Issue by JensEggers** [happner/happner-2#226]
This happened when happner-2 was upgraded from v.10 to v.11 with a previously connected browser trying to reconnect with an old token. Just need to handle the error more cleanly.

```
[ERROR] -  2827ms (Happn) __inboundLayer failure: Cannot read property 'action' of undefined at: TypeError: Cannot read property 'action' of undefined
    at ProtocolService.__cleanseRequestForLogs (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:198:15)
    at ProtocolService.__handleProtocolError (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:209:32)
    at /home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:306:26
    at ProtocolService.processInboundStack (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:77:12)
    at /home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:305:10
    at /home/jens/git/smc-core-application/node_modules/async/dist/async.js:473:16
    at next (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5329:29)
    at /home/jens/git/smc-core-application/node_modules/async/dist/async.js:969:16
    at HappnLayer.__inboundLayer (/home/jens/git/smc-core-application/node_modules/happner-2/lib/system/happn.js:292:5)
    at nextTask (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5324:14)
    at next (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5331:9)
    at /home/jens/git/smc-core-application/node_modules/async/dist/async.js:969:16
    at /home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:286:9
    at nextTask (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5324:14)
    at Object.waterfall (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5334:5)
    at ProtocolService.processLayers (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:283:9)
[ERROR] -     at HappnLayer.__inboundLayer (/home/jens/git/smc-core-application/node_modules/happner-2/lib/system/happn.js:295:14)
Unhandled rejection Error: Callback was already called.
    at /home/jens/git/smc-core-application/node_modules/async/dist/async.js:966:32
    at HappnLayer.__inboundLayer (/home/jens/git/smc-core-application/node_modules/happner-2/lib/system/happn.js:296:5)
    at nextTask (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5324:14)
    at next (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5331:9)
    at /home/jens/git/smc-core-application/node_modules/async/dist/async.js:969:16
    at /home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:286:9
    at nextTask (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5324:14)
    at Object.waterfall (/home/jens/git/smc-core-application/node_modules/async/dist/async.js:5334:5)
    at ProtocolService.processLayers (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:283:9)
    at ProtocolService.processMessageInLayers (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/protocol/service.js:303:8)
    at SessionService.handleMessage (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/session/service.js:376:32)
    at SessionService.tryCatcher (/home/jens/git/smc-core-application/node_modules/bluebird/js/release/util.js:16:23)
    at SessionService.ret [as handleMessage] (eval at makeNodePromisifiedEval (/home/jens/git/smc-core-application/node_modules/bluebird/js/release/promisify.js:184:12), <anonymous>:15:23)
    at Sparky.<anonymous> (/home/jens/git/smc-core-application/node_modules/happn-3/lib/services/session/service.js:444:10)
    at Sparky.emit (events.js:210:5)
    at Sparky.EventEmitter.emit (domain.js:499:23)


```

**Comment by southbite** [comment]
fixed in happn-3 v 11.0.0.1

**Comment by JensEggers** [comment]
see https://github.com/happner/happn-3/pull/243
